### DISCUSSION TITLE
Resource Zones, Resource Layers, Resource Widgets

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#15106] [Q&A] [answered]
How can I enable **Resource Zones, Resource Layers, Resource Widgets** feature? See the screenshot below, taken from this [video](https://www.youtube.com/watch?v=g_V7ny3LxY0&t=152s) by **Lombiq**.

![image](https://github.com/OrchardCMS/OrchardCore/assets/61869142/ee130343-58f9-4c6a-9474-9b0d077f7811)

I enabled **OrchardCore.Resources**, **OrchardCore.Layers** and **OrchardCore.Widgets** on **OrchardCore 1.7.2**, but the admin menu **Design > Resources** isn't visible.

**Comment by Piedone** [comment, marked as answer]
This was a then-private feature that Dean developed: https://github.com/OrchardCMS/OrchardCore/issues/5610#issuecomment-767475742 I don't believe Dean open-sourced it at any point, and it's certainly not a feature of OC itself. You can ask Dean for more info.
